src/download.jl

```diff
@@ -0,0 +1,51 @@
+"""
+dataurl(dataset_code)
+This function takes in a dataset code of format "GADM/<country_code>"
+Returns the URL of the gpkg dataset of the country
+"""
+function dataurl(dataset_code)
+
+    # Splits GADM/IND to GADM, IND
+    dataset_provider, country_code = split(dataset_code, "/")
+    
+    # Only accepts GADM Dataset Provider
+    if dataset_provider != "GADM"
+        error("❌ Dataset Provider $dataset_provider not supported. 💡 Please try \"GADM/<Code>\".")
+    end
+    
+    @info "✅ fetching $country_code's data from GADM\n"
```

Comment:
Unicode is nice but not very professional for some potential users of the package.